### ISSUE TITLE
Deleting multiple files should work as a single delete operation

### ISSUE THREAD

**Issue by aharpervc** [atom/tree-view#558]
Currently in Windows File Explorer, you can undo the deletes with ctrl+z, but you have to undo each file individually. Compare this to deleting multiple files in Windows, where a single ctrl+z brings back all the files at once.

**Comment by 50Wliu** [comment]
I'm assuming you meant Atom's tree view when you mentioned "Windows File Explorer"?

**Comment by aharpervc** [comment]
If you multiselect `n` files and delete them all at once in Atom's tree view, this apparently results in `n` delete operations, since undoing the operations in Windows File Explorer is a one-by-one action. I'm suggesting that it would be preferable to make Atom's tree view do that same thing in a single operation.

**Comment by adamreisnz** [comment]
Not sure if related, but currently in Mac, deleting multiple files results in only a single file being deleted and the rest becoming deselected. Will create a separate ticket for it though.

**Comment by 50Wliu** [comment]
Blocked on Electron adding a function to allow mass-deletion of files.

**Comment by aioue** [comment]
@50Wliu is there an electron issue # to track?

**Comment by 50Wliu** [comment]
I don't believe so.  Please feel free to create a new Electron issue for it.

**Comment by aioue** [comment]
I know nothing about that project and am not qualified. If it is a blocker on this repo issue, it should be raised by someone with knowledge of the internals of this `tree-view` project, and the requirements of the upstream `electron` project.

I am surprised this has not been raised since the issue surfaced in 2015. May I politely suggest people with knowledge of the blocker (yourself, @maxbrunsfeld or Jason) creates an `electron` issue.

This issue affects (afaict), delete, copy, paste - most of the `tree-view` system contextual commands.
